src/CloudService/Changelog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Used the latest version of autorest to generate code
```

Comment:
Please be more specific. For example, "Supported `-EnableNoPublicIP` when creating a databricks workspace"

Reply:
Got it,thanks for your reminder.